translate/api/Google.Cloud.Translate.V3.Samples/TranslateV3BatchTranslateText.cs

```diff
@@ -0,0 +1,108 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Generated code. DO NOT EDIT!
+
+// This is a generated sample ("LongRunningRequestPollUntilComplete", "translate_v3_batch_translate_text")
+
+using CommandLine;
+// [START translate_v3_batch_translate_text]
+using Google.Cloud.Translate.V3;
+using System;
+using System.Collections.Generic;
+
+namespace Google.Cloud.Translate.V3.Samples
+{
+    public class TranslateV3BatchTranslateText
+    {
+        /// <summary>
+        /// Batch translate text
+        /// </summary>
+        public static void SampleBatchTranslateText(string inputUri, string outputUri, string projectId, string location, string sourceLanguage, string targetLanguage)
+        {
+            TranslationServiceClient translationServiceClient = TranslationServiceClient.Create();
+            // TODO(developer): Uncomment and set the following variables
```

Comment:
This is wrong. If the developer follows this TODO and uncomments the following lines of code this code won't build since the name of the parameters are the same as the name of the variables.

Reply:
Hmm, Should I consider removing these commented lines of code or make some changes so that its at least runnable?

Reply:
Yes of course. You can add default values for parameters, or you can add a method summary explaining how to get it running.
You should add a method summary anyway explaining what the test does.